quai/handler.go

```diff
@@ -156,61 +155,89 @@ func (h *handler) checkNextPrimeBlock() {
 	for {
 		select {
 		case <-checkNextPrimeBlockTimer.C:
```

Comment:
30 seconds might be too short depending on how far behind the node is. If the network is in a state where a lot of nodes are far behind, each node will be duplicating its requests and could lead to congestion.

Reply:
This is not a append requirement, this says that we download 10-20 prime blocks in the 30 sec window, the benchmarking on a network that is running a lot of tps says that we are easily able to append 10 prime blocks in 30 sec window